polaris-core/src/main/java/org/apache/polaris/core/storage/StorageLocation.java

```diff
@@ -104,4 +112,22 @@ public boolean isChildOf(StorageLocation potentialParent) {
       return slashTerminatedLocation.startsWith(slashTerminatedParentLocation);
     }
   }
+
+  private static String scheme(String location) {
```

Comment:
nit: method name

Reply:
Changed to `getScheme`

Reply:
Did this get missed?

Reply:
Sorry - seems like things didn't push :( Will push with the next set of comments.